pandas/core/generic.py

```diff
@@ -7523,8 +7544,8 @@ def resample(self, rule, how=None, axis=0, fill_method=None, closed=None,
         used to control whether to use the start or end of `rule`.
 
         >>> s = pd.Series([1, 2], index=pd.period_range('2012-01-01',
-                                                        freq='A',
-                                                        periods=2))
+        ...                                             freq='A',
+        ...                                             periods=2))
         >>> s
         2012    1
         2013    2
```

Comment:
In the following examples, when using resample, it once uses `.head()` and the other time it shows 12 months. I don't like any of them. I'd use something like resampling a year to quarters, or a quarter to months... So we can show all the data, and it's just few rows that readers can quickly check.

Reply:
I've created two examples, one sampling a year into quarters using `convention='start'`` and another sampling quarters into months using the `convention='end'`. If it's too much we can remove one of them.